democracy/views/section_comment.py

```diff
@@ -135,24 +135,26 @@ class SectionCommentSerializer(BaseCommentSerializer):
     answers = serializers.SerializerMethodField()
     creator_name = serializers.SerializerMethodField()
     content = serializers.SerializerMethodField()
+    deleted_by_type = serializers.SerializerMethodField()
 
     class Meta:
         model = SectionComment
         fields = ['section', 'language_code', 'answers', 'comment',
                   'comments', 'n_comments', 'pinned', 'reply_to', 'creator_name',
-                  'deleted', 'deleted_at'] + COMMENT_FIELDS
+                  'deleted', 'deleted_at', 'deleted_by_type'] + COMMENT_FIELDS
 
     def get_content(self, obj):
         # Hide content if comment was deleted
 
         if not obj.deleted:
             return obj.content
-
-        if obj.deleted_by_id == obj.created_by_id:
+        elif obj.deleted_by_id is not None and obj.deleted_by_id == obj.created_by_id:
             return "Kirjoittaja poisti oman viestinsä."
-
-        return f"Viesti on poistettu {obj.deleted_at.strftime('%-d.%-m.%Y %H:%M')}, " \
-               f"koska se ei noudattanut Kerrokantasi-palvelun sääntöjä https://kerrokantasi.hel.fi/info"
+        elif obj.deleted_at:
+            deleted_time = f" {obj.deleted_at.strftime('%-d.%-m.%Y %H:%M')}" if obj.deleted_at is not None else ""
+            return f"Viesti on poistettu{deleted_time}, " \
+                   f"koska se ei noudattanut Kerrokantasi-palvelun sääntöjä https://kerrokantasi.hel.fi/info"
```

Comment:
Would it still be possible to change this to use DEMOCRACY_UI_BASE_URL setting to refer to the UI?

Reply:
Good catch, I didn't realise this setting even existed.
We need to remember to also patch the hardcoded URL from the frontend translations

Reply:
Nice.